etl/data_helpers/geo.py

```diff
@@ -54,7 +51,7 @@ def _load_countries_regions() -> pd.DataFrame:
 
 @functools.lru_cache
 def _load_income_groups() -> pd.DataFrame:
-    income_groups = Dataset(DATASET_WB_INCOME)[TNAME_WB_INCOME]
+    income_groups = Dataset(LATEST_INCOME_DATASET_PATH)["wb_income_group"]
```

Comment:
We should at some point stop using these kinds of functions. If you think about it, they will, under the hood, upload the latest versions of e.g. the income groups dataset. But in the dag, the version of the income groups dataset (if it's even explicitly given) may be a different one. So we should move to use functions that always ingest the data they need.